test/datalog_tests/function.ast.expected

```diff
@@ -1,11 +1,10 @@
 typedef Alt = C0{x: bit<32>} | C1{x: bit<32>}
 typedef C = C{f1: string, f2: string}
-extern type Set<'A>
-extern type Vec<'A>
 typedef nested_t = N{field: C}
 typedef option_t<'A> = None{} | Some{value: 'A}
+extern type std.Set<'A>
```

Comment:
What happens if you (re)declare a function from std?
This may be a problem if you output a program and try to compile that one. It should not have the std definitions...

Reply:
DDlog will complain just like with any name clash. Maybe I misunderstand the issue.

Reply:
I type a program, and then I pretty-print it (is there an option?). Then I can't compile the result anymore, because it has duplicate declarations.